CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # Icinga 2 | (c) 2012 Icinga GmbH | GPLv2+
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.17.5)
 set(BOOST_MIN_VERSION "1.66.0")
 
 if("${CMAKE_VERSION}" VERSION_LESS "3.8") # SLES 12.5
```

Comment:
Hint: when raising the minimum version, it's also a good idea to check for obvious things that are made obsolete by this. Keeping compatibility code for CMake older than 3.7 makes little sense when requiring 3.17.